src/microbe_stage/editor/CellEditorComponent.Callbacks.cs

```diff
@@ -294,8 +294,9 @@ private void DoOrganelleUpgradeAction(OrganelleUpgradeActionData data)
     {
         data.UpgradedOrganelle.Upgrades = data.NewUpgrades;
 
-        // Uncomment when upgrades can visually affect the cell
-        // UpdateAlreadyPlacedVisuals();
+        microbeVisualizationOrganellePositionsAreDirty = true;
```

Comment:
It might be good to set this to true only when the upgraded organelle has a new model path.

Reply:
That might be a bit more complicated; it would need to check out of all available upgrades for the organelle any that are being currently applied and have model path.

Reply:
I think this should just call the `OnOrganellesChanged` method as upgrades will eventually be able to impact most things. I think especially directly setting `microbeVisualizationOrganellePositionsAreDirty` to true should be avoided here as that feels hacky.

Reply:
Without setting `microbeVisualizationOrganellePositionsAreDirty` to true, game doesn't always update visuals even if upgrade changed models.

Reply:
Okay, I checked the code and it looks like the organelle add actions also set that to true, and so yeah turns out that is the intended usage, and using that'll be fine here after all. Still I'd like to see call to OnOrganellesChanged to prepare for the future where organelle upgrades might affect things like species survivability in auto-evo.

Reply:
I think it does 2 lines below